core/src/main/scala/org/apache/spark/partial/BoundedDouble.scala

```diff
@@ -21,5 +21,23 @@ package org.apache.spark.partial
  * A Double value with error bars and associated confidence.
  */
 class BoundedDouble(val mean: Double, val confidence: Double, val low: Double, val high: Double) {
-  override def toString(): String = "[%.3f, %.3f]".format(low, high)
+  override def toString(): String =
```

Comment:
I definitely can put it back, but the previous toString was just weird - it
only printed the bounds. Anyway, I'll update this in a sec (to go back).
Let me know if you change your mind.

On Saturday, April 2, 2016, Sean Owen <notifications@github.com> wrote:

> In core/src/main/scala/org/apache/spark/partial/BoundedDouble.scala
> <https://github.com/apache/spark/pull/12016#discussion_r58301669>:
>
> > @@ -21,5 +21,23 @@ package org.apache.spark.partial
> >   * A Double value with error bars and associated confidence.
> >   */
> >  class BoundedDouble(val mean: Double, val confidence: Double, val low: Double, val high: Double) {
> > -  override def toString(): String = "[%.3f, %.3f]".format(low, high)
> > +  override def toString(): String =
>
> OK, I think this is all good, except I think the toString should be left
> alone. I forgot to mention this. Not that I really expect anyone to depend
> on the format, but let's leave it since it's a public class.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/12016/files/5e3c47762f79b89544360c383db10b3d77411109#r58301669>
>

-- 
Want to work at Handy? Check out our culture deck and open roles 
<http://www.handy.com/careers>
Latest news <http://www.handy.com/press> at Handy
Handy just raised $50m 
<http://venturebeat.com/2015/11/02/on-demand-home-service-handy-raises-50m-in-round-led-by-fidelity/> led 
by Fidelity